packages/core/addon/models/delivery-rule.ts

```diff
@@ -47,8 +48,8 @@ export default class DeliveryRuleModel extends Model.extend(Validations) {
   @fragment('fragments/scheduling-rules', { defaultValue: () => ({ mustHaveData: false }) })
   schedulingRules!: SchedulingRuleFragment;
 
-  @attr()
-  format!: string;
+  @fragment('fragments/delivery-format')
+  deliveryFormat!: DeliveryFormatFragment;
```

Comment:
The backend field name is `format`

https://github.com/yavin-dev/framework/blob/4cd0d8f5353dd8c2d0c05f440e9add32719dc8b7/packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/DeliveryRule.kt#L84